infrastrucutre/iam.tf

```diff
@@ -80,7 +80,7 @@ module "iam_irsa-alb-controller" {
   oidc_providers = {
     main = {
       provider_arn               = module.eks.oidc_provider_arn
-      namespace_service_accounts = ["default:alb-controller"]
+      namespace_service_accounts = ["alb-namespace:alb-controller"]
```

Comment:
As best practice try to avoid using "namespace" in namespace name, it's clear what it's about. :)